index.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 const UNITS = ['B', 'kB', 'MB', 'GB', 'TB', 'PB', 'EB', 'ZB', 'YB'];
 
-module.exports = num => {
+module.exports = (num, locale = undefined) => {
```

Comment:
I imagined a boolean option like `localize` which enables `.toLocaleString()`.

Reply:
I want to pass the locale, because I need to enable the user to change the language during runtime in my ionic-app.
What do you think about using your approach (boolean option `localize`, default to `false`) plus adding the possibility to set the `locale` (default: undefined) to use in case of `localize` set to `true`? (This is how [moment.js does it](https://momentjs.com/docs/#/i18n/changing-locale/))

Example for default locale `de`:
```
> prettybytes(1230000);
1.23 MB
> prettybytes(1230000, true); // Default system locale 'de' is used
1,23 MB

>  prettybytes.locale('rn'); // Changes the locale for all subsequent calls with localize = true;
>  prettybytes(1230000, true): // Locale 'rn' is used
1.23 MB
```